src/interaction/VerletListVSphereInteractionTemplate.hpp

```diff
@@ -127,9 +127,7 @@ namespace espressopp {
         if(potential._computeForce(force, fsi, fsj, p1, p2)) {
           p1.force() += force;
           p2.force() -= force;
-          p1.fradius() = fsi;
```

Comment:
I implement this function according with Eq.(24) in "Flactuating soft-sphere approach to coars-graining of polymer melts, Soft matter, 2010, 6, 2282" and V_{nb}(r_{ij}) in "Equilibration of High Molecular Weight Polymer Melts: A Hierarchical Strategy, ACS Macro Lett. 2014, 3, 198".

In these paper, this function only affect the position of softblobs. The distribution of radii of gyration, which is calculated in integrator/VelocityVerletOnRadius.cpp, is represented by the function described in interaction/VSphereSelf.hpp.
Hence, I deleted propertis p1.fradius() and p2.fradius().